src/inmanta/logging.py

```diff
@@ -550,6 +523,7 @@ def __init__(self, stream: TextIO = sys.stdout) -> None:
         self._handlers: abc.Sequence[logging.Handler] = self._apply_logging_config(log_config)
         self._options_applied: bool = False
         self._logging_configs_extensions: list[LoggingConfigExtension] = []
```

Comment:
This is no longer used afaics. I don't remember what we discussed about the logging config for lsm, maybe it is still relevant to keep this around then.
```suggestion

```